src/mugs.js

```diff
@@ -679,7 +783,7 @@ define([
             },
             rawControlAttributes: {
                 presence: 'optional',
-                lstring: "Extra Control Attributes"
+                lstring: "Extra Control Attributes",
```

Comment:
I’m so conflicted on the best practice now. Is it better to maintain consistency with existing code (and omit trailing commas) or add them because it's more convenient? I think I'm tending to do whatever feels right at the time, which ironically feels wrong when I think about it in a broader sense.

These were added incidentally as the copy/paste plugin was developed (more spec attributes were added here, and then subsequently removed). I probably should have removed the commas as well just to reduce noise in the PR.